src/polyfillstub/intersection-observer-stub.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview
```

Comment:
Does this need to be a separate file from src/polyfills/intersection-observer.js? Following the code flow between the three files is a bit confusing.

Reply:
It is a bit confusing, I agree. I did this because otherwise `src/polyfills` are removed from compiler in many cases. I'd have to instead add a lot of overrides for js and mjs. And I figured that would be more dangerous rather than having a clear line between stubs and polyfill installers. But lmk if you think it'd still be better to live with exceptions.